### PR TITLE
Integrate

### LINKED ISSUE #23: Integrate runbench functionality from runbechsimon.sh in root into runbench.sh in repo dir
Runscriptsimon.sh has better comments, and implement further relevant calls to scripts that starts and stops the power plug logging on the raspberry pi, along with a call to send_results, which transfer the contents of the results folder to the raspberry pi after the benchmarks have run.

### PR DESCRIPTION
**Pull request by Villiamfj**
I hope you like shell script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,3 +79,10 @@ note: running the script from SSH requires the usage of tools like nohup:
 3. Call the benchmarking code (with administrative priviliges):
     - ``` python '.\benchmarks\fibonacci sequence\bench.py' ```
 4. The results can be found in "test.csv" in the root folder
+
+### Disabling raspberry or kill script
+The `runBench.sh` has two optinal arguments, the first is for disabling the interaction with the raspberry pi the secound is for disabling the script that stops services while running the benchmarks.
```
Comment (Jakobzs) [inline README.md]:
```suggestion
The `runBench.sh` has two optional arguments, the first is for disabling the interaction with the raspberry pi the secound is for disabling the script that stops services while running the benchmarks.
```

File: benchRunners/fib.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+bash utils/append_to_latest_csv.sh "NodeFib"
+
+#   Pypy
+pypy ./benchmarks/FibSequence/bench.py $fibInput $count
+sleep 5s
+bash utils/append_to_latest_csv.sh "PypyFib"
+
+#   C#
+# building
+dotnet build ./benchmarks/FibSequence/benchC#  --configuration Release
```
Comment (Jakobzs) [inline benchRunners/fib.sh]: Would it be possible to both build and run in one command?

File: runBench.sh [hunk 3]
```diff
@@ -1,51 +1,69 @@
...
-    timestamp=$(date +%s)
-    echo $FILE
-    mv $FILE "results/${FILE%.csv}_$1_$timestamp.csv"
-    
-}
+#!/bin/bash
 
-# stopping services
-bash kill_and_burn.sh 0
+# first argument is for stopping the logging from raspberry
```
Comment (Jakobzs) [inline runBench.sh:3]: Should probably specify what the default value is

File: utils/append_to_latest_csv.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Update lasts benchmark csv to include timestamp and mv to appropriate dir
```
Comment (Jakobzs) [inline utils/append_to_latest_csv.sh]:
```suggestion
# Update last benchmark csv to include timestamp and mv to appropriate dir
```

**Review by Jakobzs — CHANGES_REQUESTED** [review]

File: runBench.sh (hunk 3 above)
Comment (Villiamfj) [reply runBench.sh:3]: i have added a comment

File: benchRunners/fib.sh (hunk 2 above)
Comment (Villiamfj) [reply benchRunners/fib.sh]: yis, been there done that

**Review by Jakobzs — APPROVED** [review]

**Pull request merged**